src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIService.kt

```diff
@@ -151,7 +155,8 @@ class CommunityAPIEndOfServiceReportEventService(
 @Service
 class CommunityAPIActionPlanEventService(
   @Value("\${interventions-ui.baseurl}") private val interventionsUIBaseURL: String,
-  @Value("\${interventions-ui.locations.submit-action-plan}") private val interventionsUIActionPlanLocation: String,
+  @Value("\${interventions-ui.locations.submit-action-plan}") private val interventionsUISubmittedActionPlanLocation: String,
```

Comment:
won't these URLs be the same? /probation-practitioner/action-plan/{id} ?

Reply:
Can consolidate into one but will there ever be the need to make them different?